src/components/CombinePopup.vue

```diff
@@ -6,7 +6,7 @@
   <form class="combine-popup__form">
     <fieldset>
       <fieldset class="control-group">
-        <legend class="control-label">Kies het thema waarvoor u scenario's wilt combineren:</legend>
+        <legend class="control-label">Kies het thema waarvoor u scenario's <span v-if="bandCounts">(n={{ featureCount }})</span> wilt combineren:</legend>
```

Comment:
bandCounts and featureCount? should be either in this setting?

Reply:
if we did not count the number of bands I don't want to show the total number of features. So this is as intended.